build.assets/charts/Dockerfile-tbot-distroless

```diff
@@ -19,3 +19,4 @@ RUN --mount=type=bind,target=/ctx dpkg-deb -R /ctx/$TELEPORT_DEB_FILE_NAME /opt/
 FROM $BASE_IMAGE
 COPY --from=teleport /opt/staging/opt/teleport/system/bin/tbot /usr/local/bin/tbot
 ENTRYPOINT ["/usr/local/bin/tbot"]
+CMD ["start"]
```

Comment:
This is not a breaking change as specifying `CMD` here just sets the default command.
Which will be overwritten by customers who use it in kubernetes, etc:

- https://goteleport.com/docs/enroll-resources/machine-id/deployment/kubernetes/#step-45-create-a-tbot-deployment

Reply:
Same with the helm chart:

- https://github.com/gravitational/teleport/blob/3d50f1278bb54f057655f5799d67a34c2e76198f/examples/chart/tbot/templates/deployment.yaml#L50-L57

Reply:
Even if it is a breaking change I'm in favor of making it anyway to unblock this - it's been annoying for a while. It _may_ still interfere with some default behavior expectations so many it's worth mentioning in the changelog note?